lib/fog/ovirt/compute/v4.rb

```diff
@@ -63,9 +63,7 @@ def ovirt_attrs(obj)
               # ignore nil values
               next if value.nil?
 
-              if key != :data_center && value.respond_to?(:href) && value.href && value.respond_to?(:id)
-                opts[key] = value.id
-              end
+              opts[key] = value.id if key != :data_center && value.respond_to?(:href) && value.href && value.respond_to?(:id)
```

Comment:
The cop in general is correct but in this one case it makes it even harder to read with all the `and` conditions.
Commenting here as a reminder to refactor this in a separate PR.